src/server.ts

```diff
@@ -162,7 +169,8 @@ export function startServer(port: number, device: string | null = null, enableCo
       await ebb.executeMotion(motion);
     },
     async postCancel(): Promise<void> {
-      await ebb.setPenHeight(Device.Axidraw.penPctToPos(0), 1000);
+      const device = Device(ebb.hardware)
+      await ebb.setPenHeight(device.penPctToPos(0), 1000);
```

Comment:
```suggestion
      // TODO: switch to pen up position
      await ebb.setPenHeight(device.penPctToPos(50), 1000);
```

This brings the pen ALL the way up, which is not great with the brushless motor because it compresses the spring all the way. Is 50% an acceptable compromise? I will have a separate PR for setting it to the actual pen up position.

Reply:
Very curious to see what your optimal values or; we ran into the same problem and don't have the magic numbers yet.